HandleNHCreateOrUpdateInstallationCallOrchestrator/handler.ts

```diff
@@ -0,0 +1,50 @@
+import { Task } from "durable-functions/lib/src/classes";
+import * as t from "io-ts";
+import * as o from "../utils/durable/orchestrators";
+
+import { CreateOrUpdateInstallationMessage } from "../generated/notifications/CreateOrUpdateInstallationMessage";
+import { ActivityBodyImpl as CreateOrUpdateActivityBodyImpl } from "../HandleNHCreateOrUpdateInstallationCallActivity";
+import { NotificationHubConfig } from "../utils/notificationhubServicePartition";
+
+export const OrchestratorName =
+  "HandleNHCreateOrUpdateInstallationCallOrchestrator";
+
+/**
+ * Carries information about Notification Hub Message payload
+ */
+export const NhCreateOrUpdateInstallationOrchestratorCallInput = t.interface({
+  message: CreateOrUpdateInstallationMessage
+});
+
+export type NhCreateOrUpdateInstallationOrchestratorCallInput = t.TypeOf<
+  typeof NhCreateOrUpdateInstallationOrchestratorCallInput
+>;
+
+interface IHandlerParams {
+  createOrUpdateActivity: o.CallableActivity<CreateOrUpdateActivityBodyImpl>;
```

Comment:
Opinionated choice: as we have a _callable version_ of the activity (more later), it makes sense to threat each activity as a dependency for the calling orchestrator. Rationale:
* explicit dependency declaration
* less boilerplate in the orchestrator's handler 
* easier to test (it's just an injected function, no need to mock Durable Functions' internals)